blueprint/templates/subcommands.go

```diff
@@ -45,15 +45,19 @@ func SubCommands() []cli.Command {
 				},
 				cli.StringFlag{
 					Name:  "run-list",
-					Usage: "A list of comma separated cookbook recipes that is run on the servers at start-up",
+					Usage: "A list of comma separated cookbook recipes that is run on the servers at start-up, i.e: --run-list wordpress,nano,1password",
```

Comment:
would be nice to give an example for an explicit selection of a recipe within a cookbook

Reply:
Done. maybe that samples can clarify